src/components/molecules/plan/plan.scss

```
@@ -1,103 +1,81 @@
+  text-align: center;
+  width: 100%;
 
   @media (max-width: $breakpoint-md) {
-    flex-direction: column;
     margin-bottom: 24px;
   }
 
-  & > .plan-daytime {
-    width: 100%;
-  }
-
-  & > .plan-standard {
-    width: 100%;
-
-    ul {
-      border: 5px solid $orange_bistre;
+  .options {
+    background-color: $orange_zorba;
+    padding: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-weight: 700;
 
-      li:nth-child(1) {
-        height: 59px;
-        background-color: $orange_bistre;
-        padding: 15px 8px;
-      }
-    }
-  }
-
-  ul {
-    list-style: none;
-    padding: 0px;
-    margin: 0px 12px;
-
-    @media (max-width: $breakpoint-md) {
-      margin-bottom: 24px;
-    }
-
-    li {
-      padding: 20px 8px;
-      text-align: center;
-      margin: -1px 0px;
-      border: 1px solid $gray_gainsboro;
-    }
-
-    li:nth-child(1) {
-      height: 69px;
+    h3 {
       color: $white;
       font-size: $fontsize-l;
+      margin: 0;
       font-weight: 700;
-      line-height: 29px;
-      background-color: $orange_zorba;
-      margin: 0px;
-      border: none;
+      text-align: center;
     }
+  }
 
-    li:nth-child(2) {
-      height: 142px;
+  .decoration {
+    .price {
       background-color: $gray_desertStorm;
+      padding: 24px 0;
+      font-weight: 700;
+      align-items: center;
+      justify-content: center;
 
-      .plan-price {
-        display: flex;
-        flex-direction: row;
-        justify-content: center;
-        align-items: flex-end;
+      h3 {
         color: $orange_bistre;
+        font-size: $fontsize-xxl;
         font-weight: 700;
-        gap: 4px;
-        margin: 0px 0px 16px;
-
-        .plan-price-mark {
-          font-size: $fontsize-l;
-          line-height: 29px;
-        }
-
-        .plan-price-yen {
-          font-size: $fontsize-xxl;
-          line-height: 58px;
-        }
+        margin: 0;
+        text-align: center;
+      }
 
-        .plan-price-mark,
-        .plan-price-yen {
-          margin: 0px;
-        }
+      .yen {
+        font-size: $fontsize-l;
+        margin-right: 4px;
       }
 
-      .text {
-        color: $black;
+      .monthly {
         font-size: $fontsize-s;
-        margin: 0px;
+        font-weight: 400;
+        color: $black;
+        margin: 0;
       }
     }
 
-    li:nth-child(n + 3) {
-      height: auto;
-      background-color: $white;
-      padding: 20px 10px;
-      overflow-wrap: break-word;
+    .detail {
+      border: 1px solid $gray_gainsboro;
+      .list {
+        padding: 0px;
+        margin: 0;
+        font-size: $fontsize-s;
+
+        .list-items {
+          list-style: none;
+          padding: 20px;
+          border-top: 1px solid $gray_gainsboro;
+          background-color: $white;
+        }
+      }
     }
   }
 }
+
+.plan:nth-child(even) {
+  outline: 5px solid $orange_bistre;
+
+  .options {
+    background-color: $orange_bistre;
+  }
```

```suggestion
  .options {
    background-color: $orange_bistre;
    padding-top: 15px;
  }
  .list-items:last-child {
    padding-bottom: 15px;
  }
```
これではみ出さないようになるかと思います！